.github/workflows/docker_publish.yml

```diff
@@ -26,192 +26,73 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.3.0
```

Comment:
Do you have to specify the full version number, or will just `v3` work? The reason I ask is that it is often nice to automatically take advantage of minor version updates without having to update the workflow file.

Reply:
Based on what I've read [here](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions) in the "Using tags for release management", it seems like `v3` should work if the author of the action follows the tip that says "Move the major version tag (such as `v1`, `v2`) to point to the Git ref of the current release".
If it doesn't work like that, there seems to be a few custom github actions that allow for automatic updating to use the most recent release of a github action.